client/main.lua

```diff
@@ -170,18 +169,20 @@ RegisterNUICallback('volumeDown', function(_, cb)
 end)
 
 RegisterNUICallback('increaseradiochannel', function(_, cb)
-    local newChannel = radioChannel + 1
-    exports['pma-voice']:setRadioChannel(newChannel)
-    exports.qbx_core:Notify(Lang:t('new_channel')..newChannel, 'success')
+    if not onRadio then return end
```

Comment:
This is technically unnecessary as you can't trigger the nui callback without being on the radio

Reply:
even if it's powered off the nui callbacks trigger 🤔

Reply:
That shouldn't be possible though aa that requires user input to change the channel

Reply:
the UI can be open without the radio being active 🤔

Reply:
Ah yeah, nvm then